README.md

```diff
@@ -298,7 +298,7 @@ run:
 
 If you have any questions we're in #docker-machine on Freenode.
 
-=======
+=========
```

Comment:
Looks like that should either be removed or turned into a horizontal rule with `-------`.